vignettes/decontX.Rmd

```diff
@@ -222,6 +222,14 @@ decontaminated.counts <- round(decontaminated.matrix)
 seuratObject <- CreateSeuratObject(decontaminated.counts)
 ```
 
+Conversely, if you have a Seurat object containing raw count matrix and would like to run decontX, simply retrieve the count matrix, convert to SingleCellExperiment, and run on decontX.
+
+```{r seuratIntegration2, eval=FALSE}
+counts <- seurat@assays$RNA@counts
```

Comment:
We generally aren't suppose to access the assays directly with "@". Do they have an accessor function like `counts(assays(seurat)$RNA)`?

Reply:
I have changed that with a getter function.